src/entities/notification/ui/notification-item.tsx

```
@@ -0,0 +1,95 @@
+      default:
+        return <Bell className="h-3 w-3 fill-yellow-500 text-yellow-500" />;
+    }
+  };
+
+  return (
+    <div
+      className={cn(
+        'hover:bg-muted/50 flex cursor-pointer items-start gap-3 border-b p-3 transition-colors',
+        !isRead ? 'bg-blue-50/50 hover:bg-blue-50/70' : 'bg-background',
+      )}
+      role="button"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)